pages/index.vue

```diff
@@ -8,10 +8,12 @@ function filterIssuesWithLabel(issues: Issue[], label: IssueStatus): Issue[] {
 }
 
 const uniqueLabels = [...new Set(mockIssues.flatMap((issue) => issue.labels))]
+
+const isInIFrame = window !== window.parent
```

Comment:
Prefer globalThis, especially since we're using Nuxt (sure, we're an SPA now, but best to do best practices and be safe than be sorry).

```suggestion
const isInIFrame = globalThis !== globalThis.parent
```

or to better explain what's going on we could do

```suggestion
const isInIFrame = globalThis.window !== globalThis.window?.parent
```

Reply:
👍 Went with the second option